### PR TITLE
[W13-4] Book Inventory

### PR DESCRIPTION
**Pull request by kennethcsj**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -302,27 +306,126 @@ image::UndoRedoActivityDiagram.png[width="650"]
...
+** Pros: Does not need to edit Quantity Class.
+** Cons: Code becomes less cohesive.
+// end::stockcommand[]
+
 // tag::dataencryption[]
-=== [Proposed] Data Encryption
+=== Password Protection
 
-_{Explain here how the data encryption feature will be implemented}_
+==== Current Implementation
```
Comment (Sri-vatsa) [inline docs/DeveloperGuide.adoc]: Maybe you can write a bit more about some ways you have kept the password secure. Some diagrams to show interaction between classes when using passwords would help :)

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -226,7 +273,7 @@ Format: `exit`
 
 === Saving the data
 
-Address book data are saved in the hard disk automatically after any command that changes the data. +
+BookInventory data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
 // tag::dataencryption[]
```
Comment (Sri-vatsa) [inline docs/UserGuide.adoc]: Good! Might want to fill the part on data encryption in the next update to documentation. 👍

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -302,27 +306,126 @@ image::UndoRedoActivityDiagram.png[width="650"]
...
+
+.Structure of the RequestStorage Component
+image::RequestStorageClassDiagram.jpg[width="800"]
+
+Likewise, this implementation is similar to 2.5. Storage component.
+
+===== Aspect: How Request command is implemented
+
+.Structure of Request (High Level Sequence Diagram)
+image::HighLevelRequestSequenceDiagram.png[width="800"]
```
[the remark this answers is not in the document]
Comment (kennethcsj) [reply docs/DeveloperGuide.adoc]: Purple represents that it came from Model. Turquoise means that it came from Events Center

**Comment by kennethcsj** [comment]
Solving CI issues

**Pull request closed without merging**